index.html

```diff
@@ -1,16 +1,60 @@
 <!DOCTYPE html>
 <html lang="en">
-
-<head>
+  <head>
     <meta charset="UTF-8" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <title>Vanilla Todo</title>
     <link rel="stylesheet" href="style.css" />
-</head>
-
-<body>
-    <div class="container"></div>
-</body>
-<script src="script.js"></script>
+  </head>
 
-</html>
\ No newline at end of file
+  <body>
+    <div class="container">
+      <!-- Title -->
+      <h1 style="border-bottom: solid; padding-bottom: 1rem">
+        오늘 해야할 일이 뭐가 있을까 ?
+      </h1>
+      <!-- Pending List Title -->
+      <div
+        class="pending"
+        style="
+          font-size: 1.3rem;
+          font-weight: bold;
+          text-align: left;
+          margin-left: 1rem;
+        "
+      >
+        해야할 것들
+      </div>
+      <!-- Pending List -->
+      <div class="pending-list">
+        <ul class="toDos" type="square"></ul>
+      </div>
+      <!-- Done List Title -->
+      <div
+        class="done"
+        style="
+          font-size: 1.3rem;
+          font-weight: bold;
+          text-align: left;
+          margin-left: 1rem;
+        "
+      >
+        다했다~
+      </div>
+      <!-- Done List -->
+      <div class="done-list">
+        <ul class="dones" type="square"></ul>
```

Comment:
클래스명을 좀 더 구체적으로 해줬으면 좋겠어요! 사실 네이밍컨벤션에 대해서는 저도 공부하는 중이라서 더 좋은 네이밍이 있다면 알려주세요!
```suggestion
        <ul class="done-list" type="square"></ul>
```